cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -75,6 +75,14 @@ bundle agent cfe_internal_setup_knowledge
       depth_search => recurse_basedir("inf"),
       perms => mog("0440", "root", $(def.cf_apache_group) );
 
+      "$(cfe_internal_hub_vars.docroot)/application/cache."
+            handle => "cfe_internal_setup_knowledge_files_doc_root_application_cache",
+            create => "true",
+            depth_search => recurse_basedir("inf"),
+            perms => mog("0660", $(def.cf_apache_user), $(def.cf_apache_group)),
+            comment => "Ensure permissions for $(cfe_internal_hub_vars.docroot)/application/cache.
```

Comment:
Funny. The comments in these blocks (several of them nearby) have the directory plus a period, because it is a sentence, but then I guess that confuses things slightly for the promiser. ;)